add pricing (#5)
rknightuk/slashpages: pages/pricing.md

---
title: pricing

> description:  a page for those that want to buy your products can know how much they cost.

Reviewer: ```suggestion
description: a page for those that want to buy your products so they can know the cost
```

nit: readability

> creator_link: https://paul.af/pricing

Reviewer: question: not sure if this should be this URL or https://paul.af/slash-pricing

Reviewer 2: Yeah for creator links I try to link to the explainer page if I can
